src/main/java/io/lettuce/core/AbstractRedisClient.java

```diff
@@ -261,6 +267,28 @@ public void removeListener(CommandListener listener) {
         return commandListeners;
     }
 
+    /**
+     * Reauthenticate all existing connections.
+     *     
+     * @since 6.2
+     */
+    abstract public void reauthInConnections();
+
+    /**
+     * Apply a {@link Consumer} of {@link Closeable} to all active connections.
```

Comment:
...to all connections that pass the {@link selector} test.

Reply:
I copied it entirely including the comment from other class but didn't delete it from there, will fix.

Reply:
you didn't fix this comment